stubs/seaborn/seaborn/external/appdirs.pyi

```diff
@@ -0,0 +1,9 @@
+from typing_extensions import Literal
```

Comment:
This directory seems to contain vendored external libraries. Are they intended to be used by end users? If not, it may be better to omit them from the stubs.

Reply:
I see at least `seaborn.external.kde.gaussian_kde` class used as a type annotation elsewhere in the code. Should I only keep this class and delete other types in the `external` sub package? It is unlikely that their presence or absence will cause any trouble.

Reply:
Let's just keep it.